### ISSUE TITLE
Can't use on Heroku. Preinstall script fails.

### ISSUE THREAD

**Issue by summerville** [elasticsales/closeio-node#8]
What's the reason for this setup? 
```javascript
"scripts": {
    "preinstall": "sudo npm i -g mocha",
    "test": "mocha"
  },
```

It causes Heroku to fails whenever you use this lib in another project:

```
npm ERR! Failed at the close.io@0.0.4 preinstall script 'sudo npm i -g mocha'.
npm ERR! This is most likely a problem with the close.io package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     sudo npm i -g mocha
npm ERR! You can get their info via:
npm ERR!     npm owner ls close.io
npm ERR! There is likely additional logging output above.
```

**Comment by philfreo** [comment]
just merged #6. does that fix?

**Comment by summerville** [comment]
Boom! That got it :)
